### PR TITLE
VTCCA-2830 resolving dvr issues due to migration, this included split…

### PR DESCRIPTION
**Pull request by DanielConnelly**
…ting routes to better manage journey's

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -45,7 +45,7 @@ featureFlags {
   ivEnabled = false
   editNameEnabled = true
   fileUploadEnabled = false
-  dvrEnabled = false
+  dvrEnabled = true
```
Comment (habeeb01) [inline conf/application.conf]: flag should be false

File: project/FrontendBuild.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ object FrontendBuild extends Build with MicroService {
     unmanagedResourceDirectories in Assets += baseDirectory.value / "app" / "assets",
     // Dont include the source assets in the dist package (public folder)
     excludeFilter in Assets := "fonts" || "tasks" || "karma.conf.js" || "tests" || "gulpfile.js*" || "js*" || "src*" || "node_modules*" || "sass*" || "typescript*" || "typings*" || ".jshintrc" || "package.json" || "tsconfig.json" || "tsd.json"
-  )  ++ JavaScriptBuild.javaScriptUiSettings
+  )  //++ JavaScriptBuild.javaScriptUiSettings
```
Comment (habeeb01) [inline project/FrontendBuild.scala]: is there a reason for disabling javascript ?

File: test/controllers/DvrControllerSpec.scala [hunk 3]
```diff
@@ -101,29 +100,31 @@ class DvrControllerSpec extends VoaPropertyLinkingSpec {
 //    status(result) mustBe OK
```
Comment (habeeb01) [inline test/controllers/DvrControllerSpec.scala]: why is this test class commented out?

File: conf/application.conf (hunk 1 above)
Comment (DanielConnelly) [reply conf/application.conf]: enabled in production.

File: test/controllers/DvrControllerSpec.scala (hunk 3 above)
Comment (DanielConnelly) [reply test/controllers/DvrControllerSpec.scala]: comment above.

**Pull request merged**